jenkins/fake-branch.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
This file is data for an integration test Ryan requested. Added an explanation inside the script.

I know you want me to move the job configs elsewhere and I promise to do that once something starts using it, which this PR does not yet do. But since I want to require the job config to live inside that elsewhere I need it in here for now.